src/routes/MyProfile.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './MyProfile.css';
+
+const MyProfile = () => {
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const rocketsKey = Object.keys(rockets);
+  const reservedRocket = [];
+  rocketsKey.forEach((rocketId) => {
+    if (rockets[rocketId].reserved) {
+      reservedRocket.push({
+        id: rocketId,
+        name: rockets[rocketId].name,
+      });
+    }
+  });
+  const missions = useSelector((state) => state.missions.missions);
+  const missionJoined = missions.filter((mission) => {
+    if (mission.member !== true) {
+      return null;
+    }
+    return { mission };
+  });
+
+  return (
+    <div className="profile-display">
+      <div className="my-mission">
+        <h2>My Missions</h2>
+        <ul className="view-missions">
+          {missionJoined.length ? (
+            missionJoined.map((missions) => (
+              <li className="reserved-p" key={missions.mission_id}>
+                {missions.mission_name}
+              </li>
+            ))
+          ) : (
+            <li>No Missions Joinned</li>
+          )}
+        </ul>
+      </div>
+      <div className="my-rocket">
+        <h2>My Rockets</h2>
+        <div className="view-rockets">
+          {reservedRocket.length ? (
+            reservedRocket.map((rocket) => (
+              <p className="reserved-p" key={rocket.id}>
```

Comment:
- [ ] Greate job rendering a list of all reserved rockets, however, you can consider using filter() instead of the map() method, kindly try to implement this functionality it's a requirement

![Screenshot_20220722_213415](https://user-images.githubusercontent.com/25227022/180560925-472d4d3f-b077-470b-8cdd-2e44bc1c9c84.png)

Reply:
fixed